Antistasi-Ultimate-Community/A3-Antistasi-Ultimate
# Trader fast travel check

## What type of PR is this?
1. [x] Bug
2. [ ] Change
3. [ ] Enhancement
4. [ ] Miscellaneous

### What have you changed and why?
Information:
Fixing an issue where player could fast travel to trader before finishing the trader quest.

- Updated publicVariable isTraderQuestAssigned on trader quest completion
- Added string ID with localisations fo when FT is prevented (translated using Google Translator)

### Please specify which Issue this PR Resolves (If Applicable).
--

### Please verify the following.

1. [x] Have you loaded the mission in LAN host?
2. [ ] Have you loaded the mission on a dedicated server?

### Is further testing or are further changes required?

1. [x] No
2. [x] Yes (Please provide further detail below.)

### How can the changes be tested?
Steps:
Should be quite straightforward but further testing won't hurt;
Start a new save and spawn trader quest with `[] remoteExec ["SCRT_fnc_trader_prepareTraderQuest", 2];`

FT to trader should be prevented until finishing the quest. After finishing the quest, check that FT works as expected. Redo find trader task and verify again that FT is not possible while trader quest is active.

********************************************************
Notes: Might cause conflicts with https://github.com/SilenceIsFatto/A3-Antistasi-Ultimate/pull/332 but let's handle those if the time comes.
